userInfo/userInfoRouter.js

```diff
@@ -0,0 +1,66 @@
+const express = require("express");
+const db = require("./userInfoModel");
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+
+const router = express.Router();
+
+const jwtSecret = "nobody tosses a dwarf!";
```

Comment:
Put secrets in environment variables so they aren't visible on github. Even though it's not as important on this project, it's a good thing to make a habit of.